django/db/migrations/state.py

```diff
@@ -498,7 +498,7 @@ def reconstruct_manager(mgr):
                 # Force this manager to be the first and thus default
                 managers_mapping[default_manager_name] = (0, models.Manager())
             # Sort all managers by their creation counter
-            for _, manager, _ in sorted(model._meta.managers):
+            for manager in model._meta.managers:
```

Comment:
This line no longer seems to perform the sort that the commend in the previous line refers to? Maybe it's the comment that's out of date now? (haven't read the rest of the code yet)